integration/configs.go

```diff
@@ -99,6 +105,9 @@ storage:
   aws:
     dynamodbconfig:
       dynamodb: {{.DynamoDBURL}}
+  bigtable:
```

Comment:
By contract, `ChunksStorageConfig` should be specular to `ChunksStorageFlags`, so if you do a change to `ChunksStorageConfig` you should do the same change to `ChunksStorageFlags` and viceversa.

However, these configs where designed to be ready to use and should not have mixed config (either the storage is AWS or BigTable). I would suggest to:
1. Rename `ChunksStorageConfig` to `ChunksStorageDynamoDBConfig`
2. Rename `ChunksStorageFlags` to `ChunksStorageDynamoDBFlags`
3. Create `ChunksStorageBigtableConfig` and `ChunksStorageBigtableFlags` with the config you want

Reply:
It seems I can get rid of this change completely because it is not used in the new test that I added.